src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -1,8 +1,5 @@
 ﻿using System;
-using System.Collections.Specialized;
 using Android.App;
-using Android.Content.Res;
```

Comment:
Because this file doesn't have any functional changes can we leave this file unchanged?